docs/app/Examples/modules/Dimmer/States/DimmerExampleDisabled.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Dimmer, Segment } from 'semantic-ui-react'
+
+const DimmerExampleDisabled = () => (
+  <Segment>
+    <Dimmer disabled />
```

Comment:
Hm, do you know what this is supposed to do?  The dimmer still is made active when I add:
```jsx
<Dimmer active disabled />
```

Though, the description says:
>A disabled dimmer cannot be activated

Perhaps this is only applicable to the jQuery plugin in SUI core?  Thoughts?